src/ctia/stores/es/query.clj

```diff
@@ -2,6 +2,34 @@
   (:require [clj-momo.lib.es.query :as q]
             [clojure.string :as str]))
 
+(def public-tlps
+  ["white" "green"])
+
+(defn find-restriction-query-part
+  [{:keys [login groups]}]
+  ;; TODO do we really want to discard case on that?
+  (let [login (str/lower-case login)
+        groups (map str/lower-case groups)]
+    {:bool
+     {:should
+      [{:terms {"tlp" public-tlps}}
+       ;; Document Owner
+       {:bool {:filter [{:term {"owner" login}}
+                        {:terms {"groups" groups}}]}}
+
+       ;; or if user is listed in authorized_users or authorized_groups field
+       {:term {"authorized_users" login}}
+       {:terms {"authorized_groups" groups}}
+
+       ;; CTIM models with TLP amber that is owned by org BAR
+       {:bool {:must [{:term {"tlp" "amber"}}
+                      {:bool {:should [{:term {"owner" login}}
```

Comment:
In the ES query, the owner does not need to belong to the entity group? It does not seem to be consistent with the `owner?` function